packages/backend/src/utils/shellJsHelper.ts

```diff
@@ -0,0 +1,22 @@
+const Module = require("module");
+
+export const applyExecWorkaround = function () {
+  const originalRequire = Module.prototype.require;
+  Module.prototype.require = function (request: string) {
+    if (request.includes("shelljs")) {
+      const shellJsModule = originalRequire.apply(this, arguments);
+      if (shellJsModule.exec) {
+        const originalExec = shellJsModule.exec;
+        shellJsModule.exec = function () {
+          if (arguments[0].startsWith("git config --get")) {
```

Comment:
what about command like "git (several spaces)   config \t --get" ?

Reply:
I have added an explanation why do we need this.
Added unit tests